src/server/pkg/lease/lease.go

```diff
@@ -0,0 +1,38 @@
+package lease
+
+import "time"
+
+// Manager manages resources via leases
+type Manager interface {
```

Comment:
Yeah it was called `LeaseManager` then golint complains that `lease.LeaseManager` stutters.

Just tried `make lint` with `Leaser` and there was no complaint.  GG.